doc/make/msys2.md

```diff
@@ -74,6 +74,14 @@ The following linux command will do that:
 cp $(ldd lean.exe | cut -f3 -d' ' | grep mingw) .
 ```
 
+However, if you plan to use this build to compile lean programs
+to executable binaries using `lake build` in normal Windows command
+prompt outside of msys2 environment you will also need to add a windows
+version clang to your path.  This is done in the `.github\workflows\ci.yml`
+by downloading `lean-llvm-x86_64-w64-windows-gnu.tar.zst`.  You will
+see how these `clang` binaries are provided for you in a .elan toolchain
+install of Lean 4.
```

Comment:
I don't really understand these four lines, and I don't think pointing to a CI configuration file to imitate is the right approach here. :-) (And why did you write `.elan` instead of just `elan`?)

Reply:
You are right, I removed those details.